packages/apollo-server-koa/src/ApolloServer.ts

```diff
@@ -0,0 +1,178 @@
+import * as Koa from 'koa';
+import * as KoaRouter from 'koa-router';
```

Comment:
Using the `middlewareFromPath` makes sense to me from SneakyMax's example sounds great, since we're not doing fine grain routing.